OCAT/client/resources/js/components/shared/services/assessment.service.js

```diff
@@ -11,4 +11,14 @@ export class AssessmentService {
             throw new Error(`${err.response.statusText} - ${err.response.data.message}`);
         }
     }
+
+    static async retrieveAll() {
+        try {
+            const assessments = await axios.get('http://localhost:4567/api/assessment/retrieve');
```

Comment:
```suggestion
            const assessments = await axios.get('config.host/api/assessment/retrieve');
```
this is generally another good use of the config.json